README.md

```diff
@@ -2,6 +2,36 @@
 
 Shared API package for in-JVM distributed tests.
 
+# Contributing
+
+Modifications to this repository should be made when the in-jvm dtest
+API-jvm dtests needs to be extended or modified. Modifications affect all
+active Cassandra branches, so any changes should be built/tested against each branch.
+
+## Installing a Local Snapshot
```

Comment:
To be honest, I'm always just running from `-SNAPSHOT` and never symlinking in `~/.m2`. I just go to `build.xml` and change the version of the used in-jvm dtest API to the new snapshot. To make reviewers lives easier, I also publish a snapshot for them so they didn't have to build it.

Reply:
Would you prefer I remove this section then? I used it only at the end when I updated my branch to the future release version before pushing. @dcapwell also mentions he uses it. Fine either way.